specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/Clusters.json

```diff
@@ -0,0 +1,533 @@
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ClusterProperties": {
+      "properties": {
+        "nextLink": {
+          "description": "The link used to get the next page of recommendations.",
+          "type": "string"
+        },
+        "clusterId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The ID associated with the cluster."
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The provisioning state of the cluster.",
+          "enum": [
+            "Creating",
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Deleting",
+            "ProvisioningAccount"
+          ],
+          "x-ms-enum": {
+            "name": "EntityStatus",
+            "modelAsString": true
+          }
+        },
+        "encryptionKeyUri": {
+          "type": "string",
+          "description": "The Key Vault key or certificate path associated with the cluster."
+        }
+      },
+      "description": "Cluster properties."
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates that the service is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "#/definitions/ErrorDetails"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "The details of the error.",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "description": "The target of the particular error.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ClusterPatchProperties": {
+      "properties": {
+        "encryptionKeyUri": {
+          "type": "string",
+          "description": "The Key Vault key or certificate path associated with the cluster."
+        }
+      },
+      "description": "Cluster patch properties."
+    },
+    "ClusterPatch": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ClusterPatchProperties",
+          "description": "Cluster properties."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        },
+        "location": {
```

Comment:
location typically isn't changeable after create. Do you support this?

Reply:
So I wanted to remove it then though it can be useful for migrations...
But on the other side i don't really know when we are going to support it so i'm removing for now